dir.props

```diff
@@ -260,6 +260,9 @@
     <NETCoreAppTestSharedFrameworkPath>$(NETCoreAppTestRootPath)shared/Microsoft.NETCore.App/$(NETCoreAppTestSharedFxVersion)/</NETCoreAppTestSharedFrameworkPath>
     <ILCFXInputFolder>$(NETCoreAppTestRootPath)ILCInputFolder</ILCFXInputFolder>
 
+    <!-- For UAP, we'll produce the layout and hard-link this into each test's directory --> 	
+    <UAPTestSharedFrameworkPath>$(NETCoreAppTestRootPath)uap-f5-layout</UAPTestSharedFrameworkPath>	
```

Comment:
NIT: Rename the property since this is NOT a shared framework folder, so something like UAPTestRuntime might work. Also, I'm not a fan of how we use dashes on the folder name, can we instead to something like: `$(NETCoreAppTestRootPath)UAPLayout` ?